modulemd/v2/include/modulemd-2.0/private/modulemd-yaml.h

```diff
@@ -421,4 +421,129 @@ modulemd_yaml_emit_document_headers (yaml_emitter_t *emitter,
                                      guint64 mdversion,
                                      GError **error);
 
+
+/* A set of macros for simple emitting of common elements */
+#define NON_EMPTY_TABLE(table) g_hash_table_size (table) != 0
+
+#define EMIT_KEY(emitter, error, value)                                       \
```

Comment:
It seems to me that this could just be called `EMIT_SCALAR()`, since there's nothing restricting it to being a map key.